src/Car.js

```diff
@@ -0,0 +1,46 @@
+import { Console } from "@woowacourse/mission-utils";
+import ERROR_MESSAGE from "./constants/errorMessage.js";
+
+class Car {
+  static MAX_NAME_LENGTH = 5;
+
+  #name;
+  #progressCount;
+
+  constructor(name) {
+    this.#validateNameEmpty(name);
+    this.#validateNameLength(name);
+
+    this.#name = name;
+    this.#progressCount = 0;
+  }
+
+  #validateNameEmpty(name) {
+    if (name === "") {
+      throw new Error(ERROR_MESSAGE.EMPTY_NAME);
+    }
+  }
+  #validateNameLength(name) {
+    if (name.length > Car.MAX_NAME_LENGTH) {
+      throw new Error(ERROR_MESSAGE.MAX_NAME);
+    }
+  }
```

Comment:
```suggestion
  #validateNameEmpty(name) {
    if (name === "") {
      throw new Error(ERROR_MESSAGE.EMPTY_NAME);
    }
  }
  
  #validateNameLength(name) {
    if (name.length > Car.MAX_NAME_LENGTH) {
      throw new Error(ERROR_MESSAGE.MAX_NAME);
    }
  }
```
정말정말정말 사소하지만 다른 메서드들처럼 중간 개행이 하나 있으면 가독성도 좋고 일관성 있는 코드가 될 것 같아요🫡